infracost-usage-example.yml

```diff
@@ -161,6 +161,15 @@ resource_usage:
     monthly_cpu_credit_hrs: 350 # Number of hours in the month where the instance is expected to burst. Only applicable with t2, t3 & t4 Instance types. T2 requires credit_specification to be unlimited.
     vcpu_count: 2 # Number of the vCPUs for the instance type. Only applicable with t2, t3 & t4 Instance types. T2 requires credit_specification to be unlimited.
 
+  aws_elastic_beanstalk_environment.my_eb_environment:
+    monthly_data_ingested_gb: 50  # Cloudwatch log stream ingest in GB
+    monthly_data_scanned_gb: 50   # Cloudwatch data scanned in GB
+    storage_gb: 100               # Cloudwatch storage used in GB
+    new_connections: 500000    # Loadbalancer newly established connections per second on average.
+    active_connections: 100000 # Loadbalancer active connections per minute on average.
+    processed_bytes_gb: 25000  # The number of bytes processed by the load balancer for HTTP(S) requests and responses in GB.
+    rule_evaluations: 10000 # Loadbalancer rule evaulations
```

Comment:
I think you're doing this because these params directly map to the sub resources. However, it's confusing to the user. I'd recommend setting them as sub yaml object properties:

```yaml
  aws_elastic_beanstalk_environment.my_eb_environment:
    cloudwatch:
      monthly_data_ingested_gb: 50  # Cloudwatch log stream ingest in GB
      monthly_data_scanned_gb: 50   # Cloudwatch data scanned in GB
      storage_gb: 100               # Cloudwatch storage used in GB
    lb:
      new_connections: 500000    # Loadbalancer newly established connections per second on average.
      active_connections: 100000 # Loadbalancer active connections per minute on average.
      processed_bytes_gb: 25000  # The number of bytes processed by the load balancer for HTTP(S) requests and responses in GB.
      rule_evaluations: 10000 # Loadbalancer rule evaulations
```

Then in the populate usage methods you'll have to pass the sub usage data to the sub resource, see my comment on `ElasticBeanstalkEnvironment.PopulateUsage` for more info.